buildRunner is deprecated

Building rules_detekt 0.4.0 raises a warning:

```
external/rules_detekt/detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/detekt/execute/Detekt.kt:13:13: warning: 'buildRunner(Array<String>, PrintStream, PrintStream): Executable' is deprecated. Don't build a runner yourself.
            buildRunner(args, outputPrinter, errorPrinter).execute()
            ^
```

What are the alternatives here?

--- a/detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/detekt/execute/Detekt.kt
+++ b/detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/detekt/execute/Detekt.kt
@@ -1,6 +1,6 @@
 package io.buildfoundation.bazel.detekt.execute
 
-import io.gitlab.arturbosch.detekt.cli.buildRunner
+import io.gitlab.arturbosch.detekt.cli.CliRunner
 import java.io.PrintStream
 
 interface Detekt {
@@ -9,8 +9,12 @@ interface Detekt {
 
     class Impl : Detekt {
 
+        private val runner by lazy { CliRunner() }
+
         override fun execute(args: Array<String>, outputPrinter: PrintStream, errorPrinter: PrintStream) {
-            buildRunner(args, outputPrinter, errorPrinter).execute()
+            runner.run(args, outputPrinter, errorPrinter).error?.let { resultError ->
+                throw resultError
+            }
         }
     }
 }

